src/components/fetchMovie/fetchMovie.jsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+
+import api from '../../utils/api/api';
+
+
+const FetchMovieData = async (movieID = '', movieName = '') => {
+  try {
+    let params = {}
+    
+    if (movieID) {
+      params.i = movieID; 
+    } else if (movieName) {
+      params.s = movieName; 
+    } 
+    
+    const response = await api.get('', { params });
+    return response.data;
+  } catch (error) {
+    console.error('Error fetching movie data:', error);
+    return null;
+  }
+};
+
+export default FetchMovieData;
```

Comment:
We shouldn't create components for fetching data from the api.

Let's create a separate api folder and add all the routes and fetch function in that folder. And then we can use those request function from there in our components.